internal/cmd/operator-sdk/cli/cli.go

```diff
@@ -94,6 +101,7 @@ func GetPluginsCLIAndRoot() (*cli.CLI, *cobra.Command) {
 			gov2Bundle,
 			gov3Bundle,
 			helmBundle,
+			commonv3.Plugin{},
```

Comment:
I would not include this for now as it is not used for anything and it will imply backwards compatibility issues if removed later.

Reply:
It is fine. It is only to show @jmrodri the possibility of something that we spoke about. That would be in its own pr with my wish/idea of adding a Plugin Contribution doc with the vision of plugins. Also, we need/can add the custom plugin provide by kB ( the declarative.go.kubebuilder.io/v1: pattern too to allow people to get advantage of this one and custom their implementation )

It allows people such as Java developers who want to create their own plugin to follow up the same base structure and then scaffold the rest with their language of preference instead of using Golang. 

c/c